keyboards/redox/keymaps/tehl/keymap_rgb.c

```diff
@@ -0,0 +1,29 @@
+#include "keymap.h"
+#include "keymap_rgb.h"
+#include "rgblight.h"
```

Comment:
This shouldn't be needed.  It should pick this up from your `keymap.h` file, which picks it up from `quantum.h`. 

 ```suggestion
```

Reply:
Yep, that works - updated, thanks!